### ISSUE TITLE
Can't put bottles(like water bottle, some potions) in bottle slot in brewing stand

### ISSUE THREAD

**Issue by good777LUCKY** [PowerNukkit/PowerNukkit#340]
# 🐞 I found a bug

I can't put any bottles(like water bottle, potions) in bottle slot in brewing stand.

### 📸 Screenshots / Videos

(I don't have videos, sorry)

### ▶ Steps to Reproduce

1. Open brewing stand UI
2. Click any bottle that can be put in bottle slot, and click(or drag) to brewing stand's bottle slot.
3. I can't put it in. It is like when I try to put dirt in bottle slot.

### ✔ Expected Behavior

You can put bottles in bottle slot.

### ❌ Actual Behavior

Can't put it.

### 📋 Debug information

* PowerNukkit version: 1.3.1.0-SNAPSHOT
* Debug link: -
* Timings link (if relevant): -
No plugins, restone tick was enabled on `nukkit.yml`.

### 💢 Crash Dump, Stack Trace and Other Files

Not a crash.

### 💬 Anything else we should know?

But I can put normally anything else like ingredient slot or blaze slot.

**Comment by joserobjr** [comment]
# Investigation
Vanilla packets when placing the bottle in the bottom slots of the brewing stand:
```java
[02:25:21 TRACE]: [SERVER BOUND]  -   /127.0.0.1:54864: ItemStackRequestPacket(requests=[com.nukkitx.protocol.bedrock.packet.ItemStackRequestPacket$Request@62ff0600])
[02:25:21 TRACE]: [CLIENT BOUND]  -   /127.0.0.1:19132: InventorySlotPacket(containerId=4, slot=1, item=ItemData(id=373, damage=0, count=1, tag=null, canPlace=[], canBreak=[], blockingTicks=0, netId=14))
[02:25:21 TRACE]: [CLIENT BOUND]  -   /127.0.0.1:19132: ItemStackResponsePacket(entries=[ItemStackResponsePacket.Response(success=true, requestId=-79, containers=[ItemStackResponsePacket.ContainerEntry(windowId=58, items=[ItemStackResponsePacket.ItemEntry(slot=0, hotbarSlot=0, count=0, stackNetworkId=0)]), ItemStackResponsePacket.ContainerEntry(windowId=10, items=[ItemStackResponsePacket.ItemEntry(slot=1, hotbarSlot=1, count=1, stackNetworkId=14)])])])
[02:25:21 TRACE]: [CLIENT BOUND]  -   /127.0.0.1:19132: UpdateBlockPacket(flags={NEIGHBORS, NETWORK}, blockPosition=(0, 50, 0), runtimeId=3338, dataLayer=0)
```

With PowerNukkit, no packets are sent.

With Cloudburst:
```java
[02:37:17 TRACE]: [SERVER BOUND]  -   /127.0.0.1:54864: InventoryTransactionPacket(legacyRequestId=0, legacySlots=[], actions=[InventoryActionData(source=InventorySource(type=CONTAINER, containerId=124, flag=NONE), slot=0, fromItem=ItemData(id=373, damage=0, count=1, tag=null, canPlace=[], canBreak=[], blockingTicks=0, netId=1), toItem=ItemData(id=0, damage=0, count=0, tag=null, canPlace=[], canBreak=[], blockingTicks=0, netId=0), stackNetworkId=0), InventoryActionData(source=InventorySource(type=CONTAINER, containerId=6, flag=NONE), slot=1, fromItem=ItemData(id=0, damage=0, count=0, tag=null, canPlace=[], canBreak=[], blockingTicks=0, netId=0), toItem=ItemData(id=373, damage=0, count=1, tag=null, canPlace=[], canBreak=[], blockingTicks=0, netId=1), stackNetworkId=0)], transactionType=NORMAL, actionType=0, runtimeEntityId=0, blockPosition=null, blockFace=0, hotbarSlot=0, itemInHand=null, playerPosition=null, clickPosition=null, headPosition=null, hasNetworkIds=false, blockRuntimeId=0)
[02:37:17 TRACE]: [CLIENT BOUND]  -   /127.0.0.1:19132: UpdateBlockPacket(flags={NEIGHBORS, NETWORK}, blockPosition=(106, 67, 66), runtimeId=896, dataLayer=0)
```

# Possible cause
An issue in the CraftingDataPacket might be making the client ignore the potion mixes definitions, which are in the end of the packet structure.

**Comment by good777LUCKY** [comment]
I have this issue again in bleeding branch https://github.com/PowerNukkit/PowerNukkit/commit/b60abd32a850fe35f98051ecbe5a8ad955df9896.
Can't put potion that have some effect like Potion of Invisibility. Also, when I brew Awkward Potion with Gunpowder, it becomes Splash Potion of Water.

**Comment by joserobjr** [comment]
> Also, when I brew Awkward Potion with Gunpowder, it becomes Splash Potion of Water.

It was fixed in b68e9b148b273cfee3bf70c5e04833762e5d3bf7

> I have this issue again in bleeding branch [b60abd3](https://github.com/PowerNukkit/PowerNukkit/commit/b60abd32a850fe35f98051ecbe5a8ad955df9896).
> Can't put potion that have some effect like Potion of Invisibility.

I'm at `bleeding-1.3.1.2-pn` I'm going to retest at `bleeding` for `1.4.0.0-PN` soon
